src/cloud-cli/tests/cli/test_project.py

```diff
@@ -353,3 +353,36 @@ def test_project_use_by_name_and_id_error(self, config: MeltanoCloudConfig):
         )
         assert result.exit_code == 2
         assert "The '--name' and '--id' options are mutually exclusive" in result.output
+
+    def test_project_add(
+        self,
+        projects: list[CloudProject],
+        httpserver: HTTPServer,
+        config: MeltanoCloudConfig,
+    ):
+        for project in projects[:3]:
+            httpserver.expect_oneshot_request(
+                f"/projects/v1/{project['tenant_resource_key']}",
+                method="POST",
+            ).respond_with_json(project)
+            result = CliRunner().invoke(
+                cli,
+                (
+                    "--config-path",
+                    config.config_path,
+                    "project",
+                    "add",
+                    "--project_name",
+                    project["project_name"],
+                    "--git_repository",
+                    project["git_repository"],
+                    "--project_root_path",
+                    project["project_root_path"],
```

Comment:
```suggestion
                    "--config-path",
                    config.config_path,
                    "project",
                    "add",
                    "--project-name",
                    project["project_name"],
                    "--git-repository",
                    project["git_repository"],
                    "--project-root-path",
                    project["project_root_path"],
```

I suggested changes to the click options but not the tested command. Sorry about that 😞